index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <link rel="stylesheet" type="text/css" href="./index.css"/>
+    <title>YouTube video player page</title>
+</head>
+<body>
+    <nav>
+        <ul class="nav-left-section">
+            <li><img src="/img/fontawesome/align-justify.svg"></li>
```

Comment:
You should use relative paths, that way you'll make sure that images' path is being properly specified.
```suggestion
            <li><img src="./img/fontawesome/align-justify.svg"></li>
```

Reply:
Fixed this